doc/classes/Image.xml

```diff
@@ -291,25 +311,28 @@
 			<argument index="0" name="path" type="String">
 			</argument>
 			<description>
-				Load an [code]Image[/code].
+				Loads an image from file [code]path[/code].
 			</description>
 		</method>
 		<method name="lock">
 			<return type="void">
 			</return>
 			<description>
+				Locks the data and prevents writing access.
 			</description>
 		</method>
 		<method name="normalmap_to_xy">
 			<return type="void">
 			</return>
 			<description>
+				Converts the image's color data to x and y directions.
```

Comment:
- I think this would be very confusing to someone who doesn't know what a normal map is, although I'm not sure what the answer is.